kafka/src/main/java/io/micronaut/configuration/kafka/processor/KafkaConsumerProcessor.java

```diff
@@ -567,25 +572,26 @@ private void configureKeyDeserializer(Argument<?> bodyArgument, ExecutableMethod
     }
 
     @SuppressWarnings({ "rawtypes", "unchecked" })
-    private void configureValueDeserializer(Argument<?> bodyArgument, DefaultKafkaConsumerConfiguration config) {
+    private void configureValueDeserializer(Optional<Argument<?>> bodyArgument, DefaultKafkaConsumerConfiguration config) {
```

Comment:
We generally try not to use `Optional` as a method parameter (though I know there are some cases in this project where that has been violated).

Can you refactor this to use the previous signature without `Optional`?